addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationPlan.java

```diff
@@ -306,17 +307,20 @@ public boolean save() throws FileNotFoundException, JsonProcessingException {
                 data.addJob(job.getData());
             }
 
-            FilterProvider filters =
-                    new SimpleFilterProvider()
-                            .addFilter(
-                                    DefaultPropertyFilter.FILTER_ID, new DefaultPropertyFilter());
-            writer.println(YAML_OBJECT_MAPPER.writer(filters).writeValueAsString(data));
+            writeObjectAsString(data);
```

Comment:
I noticed this is in a try block with no catch. Was that intentional or a mistake?

Reply:
It's a try-with-resource statement.